apps/nextjs-website/src/messages/it.json

```
@@ -226,21 +226,21 @@
             "yourAccountIsActive": "Il tuo account è attivo"
         },
         "resetPassword": {
-            "title": "Recupera password",
             "body": "Inserisci il tuo indirizzo email e ti invieremo le istruzioni per impostare una nuova password.",
-            "goBackToLogin": "Torna al login",
-            "send": "Invia",
-            "checkEmailTitle": "Controlla l'email",
-            "checkEmailStart": "Se esiste un account associato a ",
             "checkEmailEnd": "riceverai una email con un link per impostare una nuova password.",
-            "resendEmailPrompt": "Non hai ricevuto l'email? Controlla nella posta indesiderata oppure",
-            "resendEmail": "Reinvia email",
-            "wrongEmail": "L'indirizzo email è errato?",
-            "passwordSet": "Password impostata correttamente",
+            "checkEmailStart": "Se esiste un account associato a ",
+            "checkEmailTitle": "Controlla l'email",
+            "emailFieldError": "Inserisci un indirizzo email valido",
```

```suggestion
            "emailFieldError": "L’indirizzo email non è valido",
```
on Figma there is this copy